languages/python/exercises/concept/dicts/.docs/introduction.md

```diff
@@ -0,0 +1,75 @@
+A _dictionary_ is Python's primary mapping type that connects keys with _hashable values_. The looking up of keys is more efficient than searching through an array, but requires more memory. For a detailed explaination of _hashmaps_ visit [this][hashmap-blog] blog post.
```

Comment:
Will students know what __hashable values_ are? Is the "hashable" part required, or could you say `A _dictionary_ is Python's primary mapping type that connects keys with values.` and then using the `after.md` file to explain that values are _hashable values_.

Reply:
Another point: the *keys* need to be hashable, the values don't necessarily need to be.

Reply:
@ErikSchierboom 
> Will students know what __hashable values_ are? Is the "hashable" part required, or could you say `A _dictionary_ is Python's primary mapping type that connects keys with values.` and then using the `after.md` file to explain that values are _hashable values_.

I think we should just leave the _hashable_ part out for now and then indeed just mention it in `after.md`. Should we explain it or just mention it and then link some recourses about hashing and hashmaps?

Reply:
> I think we should just leave the hashable part out for now and then indeed just mention it in after.md. 

Agreed.

> Should we explain it or just mention it and then link some recourses about hashing and hashmaps?

Good question. That depends partly on how difficult it is to explain. If it would take lots of explaining, I'd definitely just link to an external resource. If you can explain it in one or two sentences, you could do that and then also link to the external resource.

Reply:
IMO we should just mention it and link some resources because I don't think it's _that_ important for the student to completely understand yet. We should definitely revisit this subject later, but I think it's a little too early now. What do you think?

Reply:
Sounds good! 👍